### DISCUSSION TITLE
p/f1210fce6cb4447da12699dfc6f176a4/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#33] [Announcements]
# p/f1210fce6cb4447da12699dfc6f176a4/

刚来日本差不多一周了，简单谈下刚来这几天遇到的各种问题，可能在接触日本好的一面之前，吾辈先接触到了坏的一面，所以可能存在偏见，后续部分观念也许会扭转。 城市-京都之前 前往日本 中提到过，来京都的原因是厌恶了大城市的生活，但吾辈还是没想到会遇到各种不方便。 人们常说京都是一种古老的城市，吾辈现在体会到了，租房里的热水器是 94 年生产的，吾辈甚至找不到使用的教程。😂 然后是吃饭，可能是因为日本人

https://blog.rxliuli.com/p/f1210fce6cb4447da12699dfc6f176a4/

**Comment by BTBbabysscare** [comment]
文具用国誉呀

**Comment by zyzhang0417** [comment]
为啥要使用「吾辈」这个词，网名吗？或者是日语？吾辈是「我们」的意思。

**Reply by rxliuli** [reply]
只是一个自称而已，无需在意

**Comment by RinChanNOWWW** [comment]
面是鴨そば嘛？京都的鸭肉面确实挺贵。上次吃了一次一碗1000jpy，然后还点了一杯抹茶芭菲又是1000jpy……

**Comment by eraycc** [comment]


**Reply by olojiang** [reply]
博主是个，斯是陋室惟吾德馨的人。确定会是交换链接型的人？你可以跟我换。。。  （等等，我想想，我好像没有 blog 。。。 手动抖机灵）

**Reply by rxliuli** [reply]
并不，事实上，吾辈确实交换友链，ref: https://blog.rxliuli.com/links/
没有回复只是还没具体去看这位博主具体写的内容，进入网站直接来一首歌曲把吾辈吓退了 emmm

**Comment by olojiang** [comment]
阿凯同学日本行，灰常的精彩啊~ 愿好

我以为，吃至少是差不多贵的，但是好像不是。（低头看看私の工资）
- 我会很快把御三家，吃到伤的~ （换位想想觉得头有点儿痛）

电子信息化的程度，让我又坚定了在“国都”继续“苟混”的决心。
- 一个发达国家，你给我来纸质办公，这我能忍？能忍，我就不是にほhじん（所以，我忍）
- 穷则独善其身达则兼济天下，中国的传统的观点，不完全对，但是有少许的慰藉

**Comment by olojiang** [comment]
@rxliuli  月更博主，兄弟们，来催更了~
